### PR TITLE
Adds support for Tox.

### PR DESCRIPTION
**Pull request by almann**
Adds tox configuration for our stated supported versions of Python.
Adds setup tutorial documentation using pyenv.
Adds ignores for tox build detritus.

Implements #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,62 @@ This package is designed to work with **Python 2.6+** and **Python 3.3+**
...
+```
+$ pyenv local 2.6.9 2.7.12 3.3.6 3.4.5 3.5.2 pypy-5.3.1
+```
+
+Assuming you have `pyenv` properly set up (making sure `pyenv init` is evaluated into your shell),
+you can now run `tox`:
+
+```
+# Run tox for all versions of python which executes py.test.
+$ tox
```
Comment (tgregg) [inline README.md]: This sentence could benefit from a comma or two. I think it means this:

"Run tox, which executes py.test, for all versions of python."

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[tox]
+envlist = py26,py27,py33,py34,py35,pypy
+
+[testenv]
+deps=-rrequirements.txt
```
Comment (tgregg) [inline tox.ini]: Where does this file live?

Comment (almann) [inline tox.ini]: It is actually in #21.  Technically this should be rebased on #21 (I didn't want to squash them into a single PR)

Comment (tgregg) [inline tox.ini]: Nevermind, found it in #21

**Comment by tgregg** [comment]
Looks good to me.

**Pull request closed without merging**